src/components/Link/Link.tsx

```diff
@@ -0,0 +1,43 @@
+import useTheme from 'hooks/useTheme';
+import React from 'react';
+
+import { linkContainer } from './Link.style';
+import { LinkTokens, getLinkTokens } from './Link.tokens';
+import { LinkProps } from './Link.types';
+import Icon from 'components/Icon';
+
+const Link = React.forwardRef<HTMLAnchorElement, LinkProps>((props, ref) => {
```

Comment:
missing component extension for external 3rd party routers, ticket criteria quoted below
```
Link should be part of the Typography family and it could use its props maybe for further expansion
Link should be router agnostic. So removal of react-router-dom is necessary.
```

Reply:
1) Since (with the current designs and tokens) Link has very specific typography (label01-03), colors (primary, inverted) and html representation (anchor element plus whatever children the user provides) tbh I don't see any semantic or technical reason why Link and Typography (component) should be connected

2) react-router-dom is not used inside the Link component. as a package it is used by other ictinus components